cmd/logs/stern.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"text/template"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/stern/stern/stern"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/utils/pointer"
+)
+
+func getSternConfig(manifest *model.Manifest, o *LogsOptions, kubeconfigFile string) (*stern.Config, error) {
+	location, err := time.LoadLocation("Local")
+	if err != nil {
+		return nil, err
+	}
+
+	includePodQuery, err := regexp.Compile(o.Include)
+	if err != nil {
+		return nil, fmt.Errorf("failed to compile regular expression from query: %w", err)
+	}
+	var excludePodQuery *regexp.Regexp
+	if o.exclude != "" {
+		excludePodQuery, err = regexp.Compile(o.exclude)
+		if err != nil {
+			return nil, fmt.Errorf("failed to compile regular expression for excluded pod query: %w", err)
+		}
+	}
+
+	labelSelector := labels.NewSelector()
+	if !o.All {
+		req, err := labels.NewRequirement(model.DeployedByLabel, selection.Equals, []string{manifest.Name})
```

Comment:
i've been reading the conversations around pipelines, thinking out loud 🤔 now the pipelines will have a deployed by label, as the pipeline uses okteto deploy? but, the logs from the installer from running a pipeline/dependency will still be needed to be sse?

Reply:
@teresaromero not sure I follow your question. `okteto logs` are for user applications, running in the developer namespace. The installer runs in the okteto namespace

Reply:
> @teresaromero not sure I follow your question. `okteto logs` are for user applications, running in the developer namespace. The installer runs in the okteto namespace

sorry i think i did not express myself:
- due to this comment https://github.com/okteto/okteto/pull/2120#issuecomment-1008680569 i was thinking, 
- if this `okteto logs` will show the logs of a user application deployed by a pipeline run at the installer? or we have some kind of limitation with the levels still? (this is from January so maybe the behaviour has changed)
- if this is the case, the `sse` logs that are now streamed to clientes from the installer, are still needed or will this be a replacement?

Reply:
@teresaromero we now have the label `dev.okteto.com/deployed-by` on every pod on every namespace

Reply:
@pchico83 I think Tere is referring to this PR: https://github.com/okteto/okteto/pull/3073 which is using sse to get the logs. Maybe we can use stern internally instead?